src/zos_console/zowe/zos_console_for_zowe_sdk/console.py

```diff
@@ -27,8 +27,8 @@ class Console(SdkApi):
        A profile in dict (json) format
     """
 
-    def __init__(self, connection: dict):
-        super().__init__(connection, "/zosmf/restconsoles/consoles/defcn", logger_name=__name__)
+    def __init__(self, connection: dict, log = False):
```

Comment:
Can the default value for this argument be updated to `True`?
```suggestion
    def __init__(self, connection: dict, log = True):
```